tests/unit/Infrastructure/DashboardBundle/Dto/AttributeFormLanguageTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright 2022 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Tests\Unit\Infrastructure\DashboardBundle\Dto;
+
+use PHPUnit\Framework\TestCase;
+use Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Dto\Language;
+
+class LanguageTest extends TestCase
+{
+    public function test_it_is_created()
+    {
+        $en = [
+                'label' => 'emailAddressAttribute',
+                'info' => 'Description is placed here'
+        ];
+        $language = Language::fromLanguage($en);
+
+        $this->assertEquals('emailAddressAttribute', $language->label);
+        $this->assertEquals('Description is placed here', $language->info);
+    }
```

Comment:
What happens when malformed data is presented to the VO? Maybe add some tests to verify this?